.gitignore

```diff
@@ -4,7 +4,7 @@
 *.lo
 .deps
 .libs
-
+*~
```

Comment:
Would you mind splitting thischange to it's own commit?

Reply:
No could do this

Reply:
what do you mean?

Reply:
Sorry, I meant YES could do this...

Reply:
Thanks